File: Scripts/loader.py
import os
import GCT
import utils.Graphics.graphicspp as Graphics

directory = GCT.GetGCTFolder()+"\\"+"Scripts\\features"

Graphics.Notification("Starting GCTV ...")

for filename in os.listdir(directory):
    if filename.endswith(".py"):
        GCT.RunScript(os.path.join(directory, filename))

File: Scripts/loader.js
const scripts = fs.readdir(GetGCTFolder() + "\\Scripts\\features")

if (scripts) {
    for (var i = 0; i < scripts.length; i++) {
        if (fs.getFileExtension(scripts[i]) === ".js") {
            RunScript("features\\" + scripts[i])
        }
    }
}
